README.md

```diff
@@ -1722,6 +1722,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rutik Bhoyar](https://github.com/Rutikab12)
   - [Rwitesh Bera](https://github.com/rwiteshbera)
   - [Ryga](https://github.com/ryga9)
+  - [Ryan Pierce](https://github.com/rep-pierce)
```

Comment:
Thanks for opening this PR! There's a minor change needed -
Your name should be above "Ryga"